internal/command/extensions/core/core.go

```diff
@@ -472,12 +473,19 @@ func setSecretsFromExtension(ctx context.Context, app *gql.AppData, extension *E
 			return err
 		}
 	} else {
-		fmt.Fprintf(io.Out, "Set one or more of the following secrets on your target app.\n")
-		for key, value := range secrets {
-			fmt.Println(key + ": " + value.(string))
+		fmt.Fprintf(io.Out, "Set the following secrets on your target app.\n")
+		var keys []string
+
+		for key := range secrets {
+			keys = append(keys, key)
 		}
-	}
+		sort.Strings(keys)
 
+		for _, key := range keys {
+			fmt.Println(key + ": " + secrets[key].(string))
```

Comment:
should it use `fmt.Fprintf(io.Out, "%s: %s\n", key, secrets[key].(string))` ?

Reply:
👍